src/services/auth.ts

```diff
@@ -21,14 +21,8 @@ export async function login(email: string, password: string) {
 
     api.defaults.headers.authorization = `Bearer ${token}`;
 
-    const responseMe = await api.get('user/me', {});
-    const { id, full_name } = responseMe.data;
-
-    const bookmarks = await getBookmarks(id);
-
-    const user = {
-      id, name: full_name, email, bookmarks: bookmarks.data,
-    };
+    const responseMe = await api.get('user/me', { params: { bookmarks: true } });
```

Comment:
O nome do relacionamento mudará em breve para [technologyBookmarks](https://github.com/ufersa/plataforma-sabia/pull/806/files#diff-c6c1b8f4ab2c2112d51d4f6d72376a651643cd7edec4355a8a731ceb8e3d90cf) pois temos também o relacionamento com os bookmarks de serviços.
```suggestion
    const responseMe = await api.get('user/me', { params: { technologyBookmarks: true, serviceBookmarks: true } });
```